claimtxman/claimtxman.go

```diff
@@ -235,9 +235,10 @@ func (tm *ClaimTxManager) monitorTxs(ctx context.Context) error {
 		receiptSuccessful := false
 
 		for txHash := range mTx.History {
+			mTxLog.Debug("Checking if tx %s is mined", txHash)
 			mined, receipt, err = tm.l2Node.CheckTxWasMined(ctx, txHash)
 			if err != nil {
-				mTxLog.Errorf("failed to check if tx %v was mined: %v", txHash.String(), err)
+				mTxLog.Errorf("failed to check if tx %s was mined: %v", txHash.String(), err)
```

Comment:
AFAIK there's no difference between `%v` and `%s` for strings, right?

Reply:
%v is the generic and %s is specific for string. As the variable is defined as .String() there is no doubt the type